app/views/vacancies/_schools.js.haml

```diff
@@ -0,0 +1,57 @@
+:javascript
+  const getSchools = () => {
+    if (document.getElementById('map').dataset.schools) {
+      return JSON.parse(document.getElementById('map').dataset.schools);
+    }
+  };
+
+  // Stops working if I change to const initMap = () => {}
```

Comment:
reason why is this file will not be transpiled in the usual way as it is inline js